lib/statistics/statistical_test/chi_squared_test.rb

```diff
@@ -1,6 +1,9 @@
 module Statistics
   module StatisticalTest
     class ChiSquaredTest
+
+      require 'matrix'
```

Comment:
We need to add this as a dependency in the `ruby-statistics.gemspec` file, as matrix as been moved to a standalone gem instead of being included in ruby.